AppHub/AppHub/AHBuild.m

```diff
@@ -37,7 +37,7 @@ - (instancetype)initWithBundle:(NSBundle *)bundle info:(NSDictionary *)info
     if ((self = [super init])) {
         _buildDescription = [info[AHBuildDataDescriptionKey] copy] ?: @"This build was downloaded from the App Store.";
         _bundle = bundle ?: [NSBundle mainBundle];
-        _compatibleIOSVersions = info[AHBuildDataCompatibleIOSVersionsKey] ?: AHShortVersionString();
+        _compatibleIOSVersions = [info[AHBuildDataCompatibleIOSVersionsKey] allValues] ?: @[AHShortVersionString()];
```

Comment:
Ah, my mistake! You're absolutely correct- you can ignore my comment :)

I think we'll just have to fix the test here: https://github.com/AppHubPlatform/apphub-ios/blob/master/AppHub/AppHubTests/AppHubTests.m#L70, the second argument to the assert is actually a dict, so we can do `allValues` on it.